Core/Debugger/DebugInterface.h

```diff
@@ -23,16 +23,8 @@
 
 struct MemMap;
 
-enum {
-	GPR_SIZE_32,
-	GPR_SIZE_64,
-};
-
-class DebugInterface
-{
+class DebugInterface {
 public:
-	virtual ~DebugInterface() {}
```

Comment:
Wait, I just added this one.  There are virtual methods so I think it's best to have one, or else have a protected one so no one can delete the interface directly.

-[Unknown]